web/pwa/screens/BeforeWork.js

```diff
@@ -194,6 +203,44 @@ export function BeforeWork({ beginNewMission, openHistory, missions }) {
     ["desc"]
   );
 
+  function shouldDisplayEmployeeSocialImpactSurvey() {
+    const surveyId = process.env.REACT_APP_SURVEY_EMPLOYEE_SOCIAL_IMPACT;
+    if (!surveyId) {
+      return false;
+    } else if (companies?.some(c => c.admin)) {
+      return false;
+    } else if (
+      !isDateBeforeNbDays(unixTimestampToDate(userInfo.creationTime), 61)
+    ) {
+      return false;
+    } else if (hasNeverSeenSurvey(userInfo.surveyActions, surveyId)) {
+      return true;
+    } else {
+      const firstActionForSurvey = firstActionDateForSurvey(
+        userInfo.surveyActions,
+        surveyId
+      );
+      if (
+        hasNotSubmittedSurvey(userInfo.surveyActions, surveyId) &&
+        firstActionForSurvey &&
+        isDateBeforeNbDays(firstActionForSurvey, 30) &&
```

Comment:
:green_circle: On peut supprimer le magic number avec une constante : 
```suggestion
        isDateBeforeNbDays(firstActionForSurvey, NB_DAYS_BEFORE_REDISPLAY) &&
```